### ISSUE TITLE
Stop sending emails

### ISSUE THREAD

**Issue by BenWhite27** [IgorBuchelnikov/ObservableComputations#13]
Same as #12 
I've reported your behaviour to GitHub for review.  Obtaining email addresses for the purpose of sending unsolicited emails goes against GitHub's Acceptable Use Policy as stated here.

https://docs.github.com/en/github/site-policy/github-acceptable-use-policies#5-information-usage-restrictions
>You may not use information from the Service (whether scraped, collected through our API, or obtained otherwise) for spamming purposes, including for the purposes of sending unsolicited emails to users or selling User Personal Information (as defined in the GitHub Privacy Statement), such as to recruiters, headhunters, and job boards.

**Comment by IgorBuchelnikov** [comment]
I did not use GitHub Api for obtaining email addresses

**Comment by BenWhite27** [comment]
> I did not use GitHub Api for obtaining email addresses. I used email addresses from public user profiles.
The Acceptable Use Policy is not limited by the use of the API it covers all means of accessing the personal information that is made available through the service.

**Comment by IgorBuchelnikov** [comment]
My emails are referring to software development. I think if some user opens his email on the site that is referring to software development, that user expects to receive emails about software development. If that user does not want to receive emails he can hide his email. Otherwise, why open the email address? So my emails are not unsolicited and not restricted by https://docs.github.com/en/github/site-policy/github-acceptable-use-policies#5-information-usage-restrictions

**Comment by BenWhite27** [comment]
I'm sorry but you're wrong, the definition of unsolicited is "not asked for", the act of signing up for a GitHub account is not asking for people to send you emails.  Some people that sign up aren't developers and GitHub is not exclusive to software development, any project is welcome, be it hardware or software. https://github.com/EVA-3D/eva-main for example.

If you want your library to gain more traction then take a look at it from my perspective.  Your commit comments lack information ("test", "works", "fix" aren't helpful), the readme.md is... long and your release information is vague.  Fix those, write some blog posts and if the library is any good it might go somewhere.  But I've got no interest in investing my employer or personal time into even looking at this more than I've done so far.

I'll leave this up to the GitHub support team to decide now as I can't clarify my position any further.  Don't expect me to reply again.  I've got better things to do.
